### PR TITLE
This fixes 618,907,908

### PR DESCRIPTION
**Pull request by enriquesouza**

### CODE REVIEW

File: VultisigApp/VultisigApp/Views/Components/Swap/SwapDetailsSummary.swift [hunk 1]
```diff
@@ -29,9 +29,9 @@ struct SwapDetailsSummary: View {
                 getSummaryCell(leadingText: "Estimated Fees", trailingText: swapViewModel.swapFeeString(tx: tx))
             }
             
-            if swapViewModel.showDuration(tx: tx) {
-                getSummaryCell(leadingText: "Estimated Time", trailingText: swapViewModel.durationString(tx: tx))
-            }
+//            if swapViewModel.showDuration(tx: tx) {
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Views/Components/Swap/SwapDetailsSummary.swift:32]: Just remove it

File: VultisigApp/VultisigApp/Views/Swap/SwapVerifyView.swift [hunk 2]
```diff
@@ -62,10 +62,10 @@ struct SwapVerifyView: View {
                 Separator()
                 getDetailsCell(for: "Estimated Fees", with: swapViewModel.swapFeeString(tx: tx))
             }
-            if swapViewModel.showFees(tx: tx) {
-                Separator()
-                getDetailsCell(for: "Estimated Time", with: swapViewModel.durationString(tx: tx))
-            }
+//            if swapViewModel.showFees(tx: tx) {
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Views/Swap/SwapVerifyView.swift:65]: remove it

File: VultisigApp/VultisigApp/Views/Components/Swap/SwapDetailsSummary.swift (hunk 1 above)
Comment (enriquesouza) [reply VultisigApp/VultisigApp/Views/Components/Swap/SwapDetailsSummary.swift:32]: Removed

File: VultisigApp/VultisigApp/Views/Swap/SwapVerifyView.swift (hunk 2 above)
Comment (enriquesouza) [reply VultisigApp/VultisigApp/Views/Swap/SwapVerifyView.swift:65]: Removed

**Pull request merged**